admin/components/Layout/DesktopSidebar.tsx

```diff
@@ -1,43 +1,51 @@
 import Image from 'next/image'
 import { classNames } from '../../lib/classNames'
-import { isCurrent, navigation } from './Navigation'
+import { isCurrent, navigations } from './Navigation'
 import { UserProfileButton } from './UserProfileButton'
+import useTranslation from 'next-translate/useTranslation'
 
 export interface DesktopSidebarProps {
   currentTitle: string
 }
-export const DesktopSidebar = ({ currentTitle }: DesktopSidebarProps) => (
-  <div className="hidden md:flex md:w-64 md:flex-col md:fixed md:inset-y-0">
-    <div className="flex-1 flex flex-col min-h-0 bg-gray-800">
-      <div className="flex-1 flex flex-col pt-5 pb-4 overflow-y-auto">
-        <div className="flex items-center flex-shrink-0 px-4">Przewodnik</div>
-        <nav className="mt-5 flex-1 px-2 space-y-1">
-          {navigation.map((item) => (
-            <a
-              key={item.title}
-              href={item.href}
-              className={classNames(
-                isCurrent(item, currentTitle)
-                  ? 'bg-gray-900 text-white'
-                  : 'text-gray-300 hover:bg-gray-700 hover:text-white',
-                'group flex items-center px-2 py-2 text-sm font-medium rounded-md',
-              )}
-            >
-              <item.icon
+
+export const DesktopSidebar = ({ currentTitle }: DesktopSidebarProps) => {
+  const { t } = useTranslation()
+
+  return (
+    <div className="hidden md:flex md:w-64 md:flex-col md:fixed md:inset-y-0">
+      <div className="flex-1 flex flex-col min-h-0 bg-gray-800">
+        <div className="flex-1 flex flex-col pt-5 pb-4 overflow-y-auto">
+          <div className="flex items-center flex-shrink-0 px-4">
+            {t('global:webpageTitle')}
+          </div>
+          <nav className="mt-5 flex-1 px-2 space-y-1">
+            {navigations().map((item) => (
+              <a
```

Comment:
W tym momencie jak wejde na `/en-GB/locations` i klikne w ktorys z tych linkow to ląduję na polskiej wersji z powrotem.

Trzeba:
1. Trzeba tutaj otoczyc `a` komponentem `Link` i przeniesc href oraz key
2. W Linku uzyc locale zgodnie z https://github.com/vinissimus/next-translate#10-how-to-change-the-language